web/templates/user_group_settings/user_group_settings.hbs

```diff
@@ -41,7 +41,7 @@
                     {{> ../settings/settings_save_discard_widget section_name="group-permissions" }}
                 </div>
 
-                {{> group_permissions can_join_group_widget_name="can_join_group" can_mention_group_widget_name="can_mention_group" can_manage_group_widget_name="can_manage_group"}}
+                {{> group_permissions can_add_members_group_widget_name="can_add_members_group" can_join_group_widget_name="can_join_group" can_mention_group_widget_name="can_mention_group" can_manage_group_widget_name="can_manage_group"}}
```

Comment:
Doesn't this add it to the edit screen too?

Reply:
Yes, it wasn't a good way to split the commits, so just combined them now.

Reply:
Yeah, that would have been my recommendation; the second commit was pretty small and thematically similar. Had meant to explicitly suggest that squash.